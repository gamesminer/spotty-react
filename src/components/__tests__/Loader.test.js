import React from "react";
import renderer from "react-test-renderer";
import Loader from "../shared/Loader";

describe("Loader component", () => {
  const props = {
    classes: {}
  };

  it("should render correctly", () => {
    const loader = renderer.create(<Loader {...props} />).toJSON();

    expect(loader).toMatchSnapshot();
  });
});
