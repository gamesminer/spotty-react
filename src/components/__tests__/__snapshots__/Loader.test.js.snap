// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render correctly 1`] = `
<div
  className="Loader-progressContainer-2"
>
  <div
    className="MuiCircularProgress-root-3 MuiCircularProgress-colorPrimary-6 MuiCircularProgress-indeterminate-5 Loader-progress-1"
    role="progressbar"
    style={
      Object {
        "height": 160,
        "width": 160,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg-8"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle-9 MuiCircularProgress-circleIndeterminate-11"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
